cpp/src/arrow/compute/CMakeLists.txt

```diff
@@ -122,7 +134,7 @@ add_arrow_compute_test(row_test
                        row/row_test.cc
                        util_internal_test.cc
                        EXTRA_LINK_LIBS
-                       arrow_compute_testing)
+                       arrow_compute_core_testing)
```

Comment:
Is this change necessary? I think as a test defined by `add_arrow_compute_test`, it relies on extra compute stuff already?

Reply:
Thanks for confirming!